cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/UserCompanyDTO.java

```diff
@@ -5,6 +5,7 @@
 
 import javax.validation.Valid;
 import java.util.Objects;
+import java.util.Set;
 
 @ApiModel( description = "Company data of a business customer" )
 public class UserCompanyDTO extends CompanyDTO
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 7 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)